Basic/APIDiorama/Assets/Common/Scripts/DioramaUtilities.cs

```diff
@@ -0,0 +1,40 @@
+using System;
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
+using UnityEngine;
+
+namespace Unity.Netcode.Samples.APIDiorama
+{
+    /// <summary>
+    /// A class of utilities used across Dioramas
+    /// </summary>
+    public static class DioramaUtilities
+    {
+        static readonly string[] s_Usernames = new string[] { "MaryDaBest", "BobTheBell", "FranklyVal", "Georgie96", "OP Morgan" };
+
+        /// <summary>
+        /// Generates a random color
+        /// </summary>
+        /// <returns>A random RGBA color</returns>
+        public static Color32 GetRandomColor() => new Color32((byte)UnityEngine.Random.Range(0, 256), (byte)UnityEngine.Random.Range(0, 256), (byte)UnityEngine.Random.Range(0, 256), 255);
+        public static string GetRandomUsername() => s_Usernames[UnityEngine.Random.Range(0, s_Usernames.Length)];
+
+        /// <summary>
+        /// Filders some 'bad words' from a string. In a production environment, consider using an external service for complex tasks like this.
+        /// </summary>
+        /// <param name="input"></param>
+        /// <returns></returns>
+        public static string FilterBadWords(string input)
```

Comment:
Oh I noticed, this is only used inside one use-case? Shouldn't we move it to that folder path since it's use-case specific?

Reply:
We could, but at the same time Sam suggested that we keep all the "non-network specific" code in this class.
I agree with his approach, especially considering this case: in a real scenario, you just want to call a 3rd party service for doing this kind of filtering, so it makes sense to have the method impemented away from your class